tasks/manage_zfs.yml

```diff
@@ -90,15 +90,18 @@
       atime: "{{ item.atime|default(omit) }}"
       compression: "{{ item.compression|default(omit) }}"
       dedup: "{{ item.dedup|default(omit) }}"
+      encryption: "{{ item.encryption|default(omit) }}"
```

Comment:
Need to add examples of these options in defaults/main.yml so they are clear

Reply:
Ironic that I missed that, given that I just did a PR to add docs from a different PR!